### PR TITLE
Implement Unicode 11 for grapheme break

### PR DESCRIPTION
**Pull request by roever**
- include ExtendedPictograpic flag in graphemebreakdata
- remove no longer used grapheme break classes
- remove obolete rule 10
- update rule 9 and 11

- the new rule 9c is not included. It is only required for CLDR, which
we don't do, yet

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -56,6 +56,9 @@ WordBreakProperty.txt:
 GraphemeBreakProperty.txt:
 	wget http://www.unicode.org/Public/UNIDATA/auxiliary/GraphemeBreakProperty.txt
 
+emoji-data.txt:
+	wget https://www.unicode.org/Public/emoji/11.0/emoji-data.txt
```
Comment (adah1972) [inline src/Makefile.am]: Should we use a version-independent path here?

File: src/graphemebreak.c [hunk 2]
```diff
@@ -91,8 +91,37 @@ static enum GraphemeBreakClass get_char_gb_class(utf32_t ch)
     return GBP_Other;
 }
 
+/**
+ * find out if a codepoint is an extended pictographic point
```
Comment (adah1972) [inline src/graphemebreak.c]: "Finds out", as per the general code conventions in this project.

File: src/graphemebreak.c [hunk 3]
```diff
@@ -91,8 +91,37 @@ static enum GraphemeBreakClass get_char_gb_class(utf32_t ch)
...
+            return true;
+    } while (min <= max);
+
+    return false;
+}
+
 /**
  * Sets the grapheme breaking information for a generic input string.
+ * It uses the extended grapheme cluster ruleset, except for rule 9c. Rule 9c only
+ * get relevant when implementing CLDR, which we don't at the moment
```
Comment (adah1972) [inline src/graphemebreak.c]: "gets", I think? Also two spaces after the full stop is preferred.

Comment (adah1972) [reply src/graphemebreak.c]: Maybe we can delete these two lines. GB9c does not seem to make into the final TR29: http://www.unicode.org/reports/tr29/.

File: src/Makefile.am (hunk 1 above)
Comment (roever) [reply src/Makefile.am]: I would like to, but I could not find a version independent path to that file

Comment (adah1972) [reply src/Makefile.am]: In that case, we would have to live with it… 😂

**Comment by adah1972** [comment]
Oops, you made the mistake of missing committing two files (`graphemebreakdata3.tmpl` and `graphemebreakdata4.tmpl`), and I made the mistake of not fully testing before the merge. Please prepare a new PR (based on my new HEAD) to add these two files.

**Comment by adah1972** [comment]
Never mind, I added them from your `graphemebreakdata.c`.

**Pull request merged**